### ISSUE TITLE
[suggestion] Profile Page and Freinds

### ISSUE THREAD

**Issue by DakuzzHype** [rileytestut/Delta#464]
hi this is my first suggestion on github im new on this anyways here is my suggestion on what i think 1.7 would be and the update is called the social stuff i guess update (dont ask my naming scheme k)

Introduction
- 
since online so yknow how whenever you go to your profile in your nintendo switch it shows your profile, if you are online or offline, what games you have been recently playing and how many hours you have spent total well here is my take on that.

Your Profile
-
on the top left corner there will be a icon which will look like a default pfp you would find on facebook but its purple when you click that it shows a bunch of stuff, it's shaped like the nintendo switch profile screen and here is what they show

- your name
- your friend code right below your username
- your status 
(gray = offline/not playing anything, green = playing something/online, blue = playing something that has to do with WFC)
- recent games (game name, console and total hours spent)
- your custom profile pic

now don't worry there are privacy options in settings that can make you choose who gets to see who anyways thats for your profile page

Friends List
-
in order to add friends you will need 2 things, a delta account (more on that later) and your friend's code (if you have any friends) and basically you go to your friends list which is right next to the profile icon in the main menu and there will be a list, friend requests and a add friend button anyways click on the add friend button and type their code and bam friend request sent and the friend must go to the friend request and accept it and bam you got a friend

so here's what you can do with a friend
1. message them
2. if a game has a friends option for WFC (for example MKDS) then both of you can queue up and bam you are playing together (though I doubt the core that delta uses supports that and would prob cause heavy modifications to the original emu core)
3. check their profile (which shows everything that your profile shows unless they have some privacy settings on)

Delta Account
-
Basically all you gotta do is go to settings then sign in/up with google, apple or whatever you use to sign in/up and bam you have access to friends list and a friend code but if you access your profile without a delta account it will just be the same but without a friend code and the option to enter the friends list doesn't exist without an account

Example Images
- 
here are some poorly made images in ms paint

![profile stuff for delta](https://github.com/user-attachments/assets/b7459915-8066-486b-bb57-4246378dec40)
![friends list for delta](https://github.com/user-attachments/assets/1ff595a8-c8c9-4421-b5ca-9615aa17cb25)

oh btw if i was supposed to post this somewhere else then tell me please
